plugins/callback/grafana_annotations.py

```diff
@@ -26,6 +26,9 @@
 from ansible.module_utils.urls import open_url
 from ansible.plugins.callback import CallbackBase
 
+import sys
+reload(sys)
+sys.setdefaultencoding('UTF8')
```

Comment:
I think the proper patch would be to use `to_text` on the variable containing the unicode characters.


FYI, the utf-8 in the file header is to define the encoding of the `grafana_annotations.py` file itself.

Reply:
> I think the proper patch would be to use to_text on the variable containing the unicode characters.

At modern times unicode chars may exist at any data, not only messages. So all the datas should be converted to unicode. This method is simplier, IMHO

> FYI, the utf-8 in the file header is to define the encoding of the grafana_annotations.py file itself.

Thank you, I'm not familiar with python )

Reply:
See this thread: https://stackoverflow.com/questions/2276200/changing-default-encoding-of-python

Since the callback is based on ansible libraries, changing the encoding can have hidden impact.